examples/with-drizzle-appdir/src/app/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { unstable_cache } from "next/cache";
+
+import { db } from "~/server/db";
+import * as schema from "~/server/db/schema";
+import { Uploader } from "./uploader";
+
+export default async function Home() {
+  const files = await unstable_cache(
+    () => {
+      return db.select().from(schema.files);
+    },
+    [],
+    // We revalidate this tag when an upload completes
+    { tags: ["files"] },
+  )();
```

Comment:
this is probably how i'd do it - but not sure what the thought is on using unstable things in examples

Reply:
Would prefer not to suggest unstable things in our examples. Though I guess drizzle might fall into that category on it's own 😂

Reply:
Ok swapped it out for a "normal react cache" - although that's probably just as experimental as the next.js one lol

Reply:
😂